packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs-slot.css

```diff
@@ -7,9 +7,12 @@ va-breadcrumbs .va-breadcrumbs-li {
 }
 
 va-breadcrumbs .va-breadcrumbs-li::after {
-  content: ' › ';
+  content: '\f054';
   display: inline-block;
+  font: 900 .75rem 'Font Awesome 5 Free';
+  line-height: 1;
   padding: 0 0.56rem;
+  vertical-align: middle;
```

Comment:
I think using fontawesome may be unnecessary for this. Did you look at using the [encoded version of the unicode character](https://unicode-table.com/en/203A/) to see if that would resolve the problem?

![image](https://user-images.githubusercontent.com/2008881/157154203-70546014-def6-468c-9abb-36ae50e94340.png)

Reply:
@bkjohnson , I had not. Just ran with the solution that was proposed in the issue ticket by the previous lead engineer on this project. If you'd prefer we just use the unicode, I'm good with that and will make the change.